docs/README.md

```diff
@@ -4,43 +4,58 @@ description: "The csi-nfs module: General Concepts and Principles."
 moduleStatus: experimental
 ---
 
-This module provides CSI that manages volumes based on `NFS`. The module allows you to create a `StorageClass` in `Kubernetes` by creating [Kubernetes custom resources](./cr.html#nfsstorageclass) `NFSStorageClass`.
+The module provides CSI for managing NFS volumes and allows creating StorageClass in Kubernetes through [Custom Resources](./cr.html#nfsstorageclass) `NFSStorageClass`.
 
-> **Caution!** The user is not allowed to create a `StorageClass` for the `nfs.csi.k8s.io` CSI driver.
+{{< alert level="info" >}}
+Creating a StorageClass for the CSI driver `nfs.csi.k8s.io` by the user is prohibited.
+{{< /alert >}}
 
 ## System requirements and recommendations
 
 ### Requirements
-- Stock kernels shipped with the [supported distributions](https://deckhouse.io/documentation/v1/supported_versions.html#linux).
-- Presence of a deployed and configured NFS server.
+
+- Use stock kernels provided with [supported distributions](https://deckhouse.io/documentation/v1/supported_versions.html#linux);
+- Ensure the presence of a deployed and configured NFS server;
+- To support RPC-with-TLS, enable `CONFIG_TLS` and `CONFIG_NET_HANDSHAKE` options in the Linux kernel.
+
+### Recommendations
+
+For module pods to restart when the `tlsParameters` parameter is changed in the module settings, the [pod-reloader](https://deckhouse.io/products/kubernetes-platform/documentation/v1/modules/pod-reloader) module must be enabled (enabled by default).
+
+## RPC-with-TLS mode limitations
+
+- Only one certificate authority (CA) is supported.
+- For the `mtls` security policy, only one client certificate is supported.
+- A single NFS server cannot simultaneously operate in different security modes: `tls`, `mtls`, and standard (non-TLS) mode.
+- The `tlshd` daemon must not be running on cluster nodes, as it will conflict with the module's `tlshd` daemon.
```

Comment:
```suggestion
- The `tlshd` daemon must not be running on the cluster nodes, otherwise it will conflict with the daemon of our module. To prevent conflicts when enabling TLS, the third-party `tlshd` is automatically stopped on the nodes and its autostart is disabled.
```